src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo list</title>
+
+  <!-- Font awesome icons -->
+  <script src="https://kit.fontawesome.com/6ce384dd4f.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <section id="card-bg">
+      <!-- Container -->
+    <div id="container">
+      <div class="heading">
+        <h2 class="list-title">Today's To Do</h2>
+        <i class="fa-solid fa-rotate reload-icon icon"></i>
+      </div>
+      <hr class="line-separator">
+
+      <!-- Input -->
+      <form action="">
+        <input class="new-task" type="text" placeholder="Add to your list..." required>
```

Comment:
You have a form without any action in your html code.   This link is a helpful tool to check your code: https://validator.w3.org/nu/#textarea

Reply:
Nice link, didn't know that existed, thank you! @AlejandraKeber  🙌